scrapy/extensions/logstats.py

```diff
@@ -30,21 +30,25 @@ def from_crawler(cls, crawler):
     def spider_opened(self, spider):
         self.pagesprev = 0
         self.itemsprev = 0
-
+        self.response_bytesprev = 0.0
         self.task = task.LoopingCall(self.log, spider)
         self.task.start(self.interval)
 
     def log(self, spider):
         items = self.stats.get_value('item_scraped_count', 0)
         pages = self.stats.get_value('response_received_count', 0)
+        response_bytes = self.stats.get_value('downloader/response_bytes', 0) / float(1000)
```

Comment:
:lipstick: I don’t think `float` is needed here, in Python 3 divisions always return a `float`.

Also, a more common way to express literals as floats is to append `.0` to them, so if this was not a division, `1000.0` would probably have been slightly more idiomatic.